apps/api-gateway/src/verification/verification.controller.ts

```diff
@@ -33,6 +33,8 @@ import { User } from '../authz/decorators/user.decorator';
 import { GetAllProofRequestsDto } from './dto/get-all-proof-requests.dto';
 import { IProofRequestSearchCriteria } from './interfaces/verification.interface';
 import { ProofRequestType, SortFields } from './enum/verification.enum';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Any specific reason to disable `no-unused-vars` ?

Reply:
Resolved